h/api/models/token.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+import uuid
+
+import sqlalchemy
+
+from h.api import db
+
+
+class Token(db.Base):
+
+    """A long-lived API token for a user."""
+
+    __tablename__ = 'token'
+    userid = sqlalchemy.Column(sqlalchemy.UnicodeText(),
+                               primary_key=True)
+    token = sqlalchemy.Column(sqlalchemy.UnicodeText(),
+                              unique=True,
+                              nullable=False)
+
+    def __init__(self, userid):
+        self.userid = userid
+        self.token = 'dev-' + str(uuid.uuid4())
```

Comment:
1. There's no particular reason to use a UUID here. 
2. Most API tokens are "developer" tokens, I think we probably want to call these something else, and probably leave the name internal.

I'd be more inclined to go with something like:

```python
'6879-' + binascii.hexlify(os.urandom(16))
```

(The type field doesn't matter, particularly, but [0x68, 0x79] == ['h', 'y'])